containers/compose.yaml

```diff
@@ -21,13 +22,13 @@ services:
     environment:
       POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
       POSTGRES_USER: ${POSTGRES_USER}
-      PGDATA: /var/lib/postgresql/data
-    user: "postgres:postgres"
+      POSTGRES_DB: ${POSTGRES_DB}
+    user: "${POSTGRES_HOST_UID}:${POSTGRES_HOST_GID}"
     volumes:
-      - ${POSTGRES_CONFIG_DIR_HOST}/postgresql.conf:/etc/postgresql/postgresql.conf:ro
-      - ${POSTGRES_DATA_DIR_HOST}:/var/lib/postgresql/data:z
+      - ${POSTGRES_DATA_DIR_HOST}/postgresql.conf:/etc/postgresql/postgresql.conf:ro
+      - ${POSTGRES_DATA_DIR_HOST}/pgdata:/var/lib/postgresql/data
```

Comment:
Maybe the data directory should be configurable in case we want to host the data on different Storage disk.